packages/components/organisms/f-footer/test/specs/component/f-footer-au.component.spec.js

```diff
@@ -0,0 +1,24 @@
+import FooterComponent from '../../../test-utils/component-objects/f-footer.component';
+const path = 'iframe.html?id=components-organisms--' // storybook url for all components - could move to config.
+const locale = '&knob-Locale=en-AU'
+const links = '&knob-Show%20courier%20links=true'
+
+describe('f-footer component tests', () => {
+    beforeEach(() => {
+        browser.url(`${path}footer-component${locale}`);
+        FooterComponent.waitForFooter();
+    });
+
+    it('Does not show courier links on en-AU if courier links is set to false', () => {
```

Comment:
Start with should? `should not show courier links on en-AU if courier links is set to false` ?